packages/debugger-extension/src/index.ts

```diff
@@ -166,19 +174,43 @@ const files: JupyterFrontEndPlugin<void> = {
 const notebooks: JupyterFrontEndPlugin<void> = {
   id: '@jupyterlab/debugger-extension:notebooks',
   autoStart: true,
-  requires: [IDebugger, INotebookTracker],
+  requires: [IDebugger, INotebookTracker, IMainMenu],
   optional: [ILabShell],
   activate: (
     app: JupyterFrontEnd,
     service: IDebugger,
     notebookTracker: INotebookTracker,
+    mainMenu: IMainMenu,
     labShell: ILabShell | null
   ) => {
+
     const handler = new Debugger.Handler({
       type: 'notebook',
       shell: app.shell,
       service
     });
+    
+    app.commands.addCommand('debugger:restart-debug', {
```

Comment:
I will improve that so that the command can be easily added to the toolbar and context menus.